src/epgsql.erl

```diff
@@ -260,6 +261,13 @@ set_notice_receiver(C, PidOrName) ->
 get_cmd_status(C) ->
     epgsql_sock:get_cmd_status(C).
 
+%% @doc Returns the OS pid of PostgreSQL backend OS process that serves this connection.
+%%
+%% Similar to `SELECT pg_get_pid()', but does not need network roundtrips.
+-spec get_os_pid(connection()) -> integer().
+get_os_pid(C) ->
```

Comment:
I find it confusing the terminology around `os` since this call will be executed from a process within the BEAM that is potentially living in another machine compared to the PG server.
How about renaming it to `pg_backend_pid`?

Reply:
Good idea. I'd prefer `get_backend_pid` though, because `epgsql` is already about postgres